tests/integration/contracts/conftest.py

```
@@ -0,0 +1,14 @@
+"""
+
+import sys
+from pathlib import Path
+
+# Add fixtures to path
+fixtures_path = Path(__file__).parent.parent / "fixtures"
+if str(fixtures_path) not in sys.path:
+    sys.path.insert(0, str(fixtures_path))
+
+# Import main configuration
+from conftest import *
```

Ruff (0.13.1)

`from conftest import *` used; unable to detect undefined names

(F403)